tests/app/letters/test_returned_letters.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 
 
+@pytest.mark.skip(reason="Deprecated: LETTER CODE")
```

Comment:
Just curious why you are marking some things for deprecation and removing others?  Just the process you are following?

Reply:
this is good question! I basically was only concentrating on app/tasks.py and test_tasks.py. These two were the instigator. Now if the code from them was called else where I was deleting that code as well. In the above case - the actual rest API hasn't been removed but as it was mocking part of the tasks celery code, we were causing a failure. When the rest API is removed, I was thinking we can remove these further tests.

Reply:
Makes sense, thanks for the clarification.